jobs/service-fabrik-apiserver/templates/config/bpm.yml.erb

```diff
@@ -1,7 +1,29 @@
 ---
 processes:
 - name: service-fabrik-apiserver
-  executable: /var/vcap/jobs/service-fabrik-apiserver/bin/service-fabrik-apiserver
-  persistent_disk: true
-  limits:
-    open_files: 100000
```

Comment:
Don't we want to keep the open file descriptors limit?

Reply:
bpm internally has a limit of open files. Don't want to override that.

Reply:
Ok... fine